packages/SwingSet/test/devices/test-devices.js

```diff
@@ -64,7 +64,16 @@ test.serial('d0', async t => {
   const hostStorage = provideHostStorage();
   await initializeSwingset(config, [], hostStorage);
   const c = await makeSwingsetController(hostStorage, {});
+  for (let i = 0; i < 5; i += 1) {
```

Comment:
wow these tests are fussy

I'm inclined to do `c.run()` in as many of them as possible, rather than carefully (and probably fragily) doling out a specific number of steps ahead of time. This test doesn't look like it does any additional steps, so I think it might work (and be cleaner) to `await c.run()` instead.

Reply:
I went through all the tests and converted the ones that could succeed with `c.run()` to do so.  That was maybe half of them or a little more.  I'm not quite sure what the issues were in the places where it didn't work, but I don't think it's worth trying to figure that out just now.